modules/percolator/src/test/java/org/elasticsearch/percolator/PercolatorFieldMapperTests.java

```diff
@@ -112,6 +112,7 @@
 import static org.hamcrest.Matchers.equalTo;
 import static org.hamcrest.Matchers.instanceOf;
 
+//TODO migrate tests that don't require a node to a unit test that subclasses MapperTestCase
 public class PercolatorFieldMapperTests extends ESSingleNodeTestCase {
```

Comment:
good to know. It looked easy at first sight, at least for the testEmpyName method that I had to manually correct, but I guessed I was missing something. It may be a case of splitting it into two tests, but it was not entirely clear to me how many tests we can move to the unit tests class.